extensions/amp-connatix-player/0.1/test/test-amp-connatix-player.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../amp-connatix-player';
+
+describes.realWin('amp-connatix-player', {
+  amp: {
+    extensions: ['amp-connatix-player'],
+  },
+}, env => {
+
+  let win;
+  let doc;
+
+  beforeEach(() => {
+    win = env.win;
+    doc = win.document;
+  });
+
+  function getConnatixPlayer(attributes) {
+    const cnx = doc.createElement('amp-connatix-player');
+    for (const key in attributes) {
+      cnx.setAttribute(key, attributes[key]);
+    }
+    cnx.setAttribute('width', '480');
+    cnx.setAttribute('height', '270');
+    cnx.setAttribute('layout', 'responsive');
+
+    doc.body.appendChild(cnx);
+    return cnx.build().then(() => { cnx.layoutCallback(); return cnx; });
+  }
+
+  it('renders', async() => {
```

Comment:
Very nice using `async/await`.

Reply:
@alanorozco , now I also made the unlayoutCallback test async for consistency :)